src/Libraries/PythonNodeModelsWpf/ScriptEditorWindow.xaml.cs

```diff
@@ -105,6 +105,47 @@ internal void Initialize(Guid workspaceGuid, Guid nodeGuid, string propName, str
             CachedEngine = nodeModel.EngineName;
             EngineSelectorComboBox.SelectedItem = CachedEngine;
         }
+
+        #region Text Zoom in Python Editor
+        // React to ctrl + mouse wheel to zoom in and out
+        private void EditorBox_PreviewMouseWheel(object sender, MouseWheelEventArgs e)
+        {
+            bool ctrl = Keyboard.Modifiers == ModifierKeys.Control;
+            if (ctrl)
+            {
+                this.UpdateFontSize(e.Delta > 0);
+                e.Handled = true;
+            }
+        }
+
+        // Reasonable max and min font size values
+        private const double FONT_MAX_SIZE = 60d;
+        private const double FONT_MIN_SIZE = 5d;
+
+        // Update function, increases/decreases by a specific increment
```

Comment:
1. This comment doesn't really make sense.
2.  public methods should use ///summary comments.
3. Why is this public?

Reply:
@mjkkirschner it's code from elsewhere ported. For sure not ideal for Dynamo. @QilongTang is fixing it for me 😊

Reply:
Oh missed this one, I will update this in another PR